### PR TITLE
[#118458297] Postgres single AZ plan

### PR DESCRIPTION
**Pull request by alext**
## What

This adds the first real service plan for postgres. This is a single AZ plan (not highly available). Other plans will follow in subsequent stories.

This additionally makes the following changes:

* Removes the temporary test plan that was in place solely to validate the broker.
* Update the trade-tariff quota to allow non-free plans
* Refactor the quota-setting task to me more robust, and have the logic in a single place.

## How to review

Deploy from this branch.  Verify that the new plan is available, and that the description matches what's specified in the spreadsheet linked from [the pivotal story](https://www.pivotaltracker.com/story/show/118458297).
Verify that the custom acceptance tests pass.

## Who can review

Anyone but myself.

### CODE REVIEW

File: concourse/scripts/set_quotas_from_manifest.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+require 'json'
+require 'yaml'
+
+class QuotasSetter
+  def initialize(manifest)
+    @manifest = manifest
+  end
+
+  def apply!
+    cc_properties = @manifest.fetch("properties").fetch("cc")
```
Comment (dcarley) [inline concourse/scripts/set_quotas_from_manifest.rb]: Rubocop failed the build:
```
concourse/scripts/set_quotas_from_manifest.rb:12:5: W: Useless assignment to variable - cc_properties.
    cc_properties = @manifest.fetch("properties").fetch("cc")
    ^^^^^^^^^^^^^
```

File: manifests/cf-manifest/deployments/060-aws-broker.yml [hunk 2]
```diff
@@ -62,6 +62,33 @@ jobs:
                     db_subnet_group_name: (( grab terraform_outputs.rds_broker_dbs_subnet_group ))
                     vpc_security_group_ids:
                       - (( grab terraform_outputs.rds_broker_dbs_security_group_id ))
+                - id: "9b882524-ab58-4c18-b501-d2a3f4619104"
+                  name: "M-dedicated-9.5"
+                  description: "20GB Storage, Dedicated Instance, 20 Concurrent Connections. Postgres Version 9.5"
```
Comment (mtekel) [inline manifests/cf-manifest/deployments/060-aws-broker.yml]: I think the server should be able to take _many more_ than 20 concurrent connections...

File: concourse/scripts/set_quotas_from_manifest.rb (hunk 1 above)
Comment (alext) [inline concourse/scripts/set_quotas_from_manifest.rb]: Doh, copy/paste error...  Fixed.

File: manifests/cf-manifest/deployments/060-aws-broker.yml (hunk 2 above)
Comment (alext) [inline manifests/cf-manifest/deployments/060-aws-broker.yml]: This plan is an initial best guess based on the trade-tariff requirements. These will be refined over time as we gain a better understanding of our tenants' needs.

File: concourse/scripts/spec/set_quotas_from_manifest_spec.rb [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  let(:manifest) {{
+    "properties" => {
+      "cc" => {
+        "quota_definitions" => quota_definitions,
+      },
+    },
+  }}
+  subject { described_class.new(manifest) }
+
+  before :each do
```
Comment (saliceti) [inline concourse/scripts/spec/set_quotas_from_manifest_spec.rb]: This is not strictly necessary as we always override it below.

**Comment by saliceti** [comment]
Confusing tipo in 88a23b7 commit message: No what -> Now that?

**Pull request closed without merging**